app/controllers/posts_controller.rb

```diff
@@ -2,7 +2,7 @@ class PostsController < ApplicationController
   # Get users/:user_id/posts
   def index
     @user = User.find(params[:user_id])
-    @posts = Post.where(author_id: params[:user_id])
+    @posts = Post.includes(:comments).where(author_id: params[:user_id])
```

Comment:
- [ ] @Team Great work fixing the N + 1 problem however the screenshots provided don't show your app's logs before and after the fix, kindly consider indicating the before and after logs screenshots